lib/features/app/di/app_scope.dart

```diff
@@ -142,5 +141,5 @@ abstract class IAppScope {
   SharedPreferences get sharedPreferences;
 
   /// Analytics sending service
-  IAnalyticsService get analyticsService;
+  AnalyticService get analyticsService;
```

Comment:
It is better to set type `AnalyticActionPerformer` so that you cannot add a new strategy to `AnalyticService` anywhere in the application